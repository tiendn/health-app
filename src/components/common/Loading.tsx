import React from "react";
import styles from "./style.module.scss";

const Loading = () => {
	return (
		<div className={styles.loadingSpinner}>
			<div className={styles.spinner}></div>
		</div>
	);
};

export default Loading;
