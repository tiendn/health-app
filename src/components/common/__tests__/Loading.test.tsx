import React from "react";
import renderer from "react-test-renderer";
import Loading from "../Loading";

test("Badge renders correctly", () => {
	const tree = renderer.create(<Loading />).toJSON();
	expect(tree).toMatchSnapshot();
});
