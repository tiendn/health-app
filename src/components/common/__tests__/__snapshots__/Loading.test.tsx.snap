// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Badge renders correctly 1`] = `
<div
  className="loadingSpinner"
>
  <div
    className="spinner"
  />
</div>
`;
